doc/source/whats_new.rst

```diff
@@ -9,6 +9,10 @@ Current
 Changelog
 ~~~~~~~~~
 
+   - Add support for single dipole fitting by `Eric Larson`_
+
+   - Add support for spherical BEM models by `Eric Larson`_
```

Comment:
again spherical and BEM in one sentence does not make sense to me. BEM refers to boundary element method and numerical integration. I would just say

Add support for spherical forward models by `Eric Larson`_